spec/acceptance/nodesets/default.yml

```diff
@@ -1,22 +1,32 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
 HOSTS:
   server-el7:
     roles:
       - server
       - default
       - master
       - el7
-    platform:   el-7-x86_64
-    box:        centos/7
+    platform: el-7-x86_64
+    box: centos/7
     hypervisor: vagrant
 
   server-el6:
     roles:
       - el6
-    platform:   el-6-x86_64
-    box:        centos/6
+    platform: el-6-x86_64
+    box: centos/6
     hypervisor: vagrant
 
 CONFIG:
   log_level: verbose
-  type:      aio
+  type: aio
   vagrant_memsize: 256
+<% if ENV['BEAKER_PUPPET_ENVIRONMENT'] -%>
+  puppet_environment: <%= ENV['BEAKER_PUPPET_ENVIRONMENT'] %>
```

Comment:
* `BEAKER_PUPPET_ENVIRONMENT` should be `BEAKER_PUPPET_COLLECTION`
* `puppet_environment` should be `puppet_collection`

See [SIMP-5430][SIMP-5430] and pupmod-simp-tlog's [`.gitlab-ci.yml`][0]

[0]: https://github.com/simp/pupmod-simp-tlog/blob/master/.gitlab-ci.yml#L72-L91
[SIMP-5430]: https://simp-project.atlassian.net/browse/SIMP-5430

Reply:
Will fix